tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -723,6 +723,30 @@ module.exports = {
       return this
     },
 
+    cancelMoveResource: async function(target) {
+      await this.waitForFileVisible(target)
+
+      // cancel move
+      await this.useXpath()
+        .waitForElementVisible(client.page.filesPage().elements.cancelMoveCopyBtn.selector)
+        .click(this.page.filesPage().elements.cancelMoveCopyBtn.selector)
+        .useCss()
+
+      return this
+    },
+
+    cancelCopyResource: async function(target) {
+      await this.waitForFileVisible(target)
+
+      // cancel copy
+      await this.useXpath()
+        .waitForElementVisible(client.page.filesPage().elements.cancelMoveCopyBtn.selector)
+        .click(this.page.filesPage().elements.cancelMoveCopyBtn.selector)
+        .useCss()
+
+      return this
+    },
```

Comment:
```suggestion
    cancelResourceMoveOrCopyProgress: async function(target) {
      await this.waitForFileVisible(target)

      // cancel move
      await this.useXpath()
        .waitForElementVisible(client.page.filesPage().elements.cancelMoveCopyBtn.selector)
        .click(this.page.filesPage().elements.cancelMoveCopyBtn.selector)
        .useCss()

      return this
    },
```
Be more [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself#:~:text=The%20DRY%20principle%20is%20stated,their%20book%20The%20Pragmatic%20Programmer.). It seems a single function is enough for this type of action!